### DISCUSSION TITLE
Clients

### DISCUSSION THREAD

**Discussion by JohnFawkes** [AdguardTeam/AdGuardHome#4447] [Q&A]
So since I'm on Sprint, we'll T-mobile now, they only use ipv6 addresses. Is there a way for me to allow requests from my device using adguard app on android to allow my phone to connect to my adguardhome? Everything I've tried gives me no connection error in adguard when I try to add it via dns in the app. This ipv6 address is not static either btw.

**Comment by MembershipExcel** [comment]
This might not be the solution you are looking for but did you try nextdns ?

**Reply by JohnFawkes** [reply]
not exactly what i want. I want to use the adguard app and I can connect with my it, the issue is that I use the allowlist to only allow certain IPs to connect to adguard. Using the clientid I can connect just fine doh but not with quic or tls

**Comment by ainar-g** [comment]
You may solve it by using some kind of Dynamic DNS service.  Or your router's DNS, although that would only work within the router's network.  But generally such problems are solved by hosting your server somewhere with a static IP address and a hostname.
